packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -0,0 +1,213 @@
+# Nimble Wafer Map
```

Comment:
Heads up that the PR pipeline is failing because of lint errors in this new file. Before you submit you can run `npm run format` from the root of the repo to fix this (after running `npm install` to install the lint tooling of course).